packages/crud/src/vaadin-lit-crud-edit.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2024 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See https://vaadin.com/commercial-license-and-service-terms for the full
+ * license.
+ */
+
+import { html } from 'lit';
+import { Button } from '@vaadin/button/src/vaadin-lit-button.js';
+import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+/**
+ * Use registerStyles instead of the `<style>` tag to make sure
+ * that this CSS will override core styles of `vaadin-button`.
+ */
+registerStyles(
+  'vaadin-crud-edit',
+  css`
```

Comment:
I think this can be replaced with `static get styles()` as we generally do in Lit versions. 

IIRC the original issue was concerning Polymer version of `vaadin-crud-edit` back in the day when `vaadin-button` was still using `<style>` tag in template, so the comment can be probably removed from there as well.

Reply:
Done